verify_load_tests/main.go

```diff
@@ -0,0 +1,68 @@
+package main
```

Comment:
nit: I'd probably put this under the existing `hack` directory rather than create a new dir for it to keep things cleaner, not a big deal either way.